ts/store/reducers/entities/organizations/organizationsAll.ts

```diff
@@ -25,6 +26,12 @@ const reducer = (
   action: Action
 ): OrganizationsAllState => {
   switch (action.type) {
+    // when this action is performed, all elements in which the value of the fiscalCode
+    // field is not present in the payload are removed from the state array.
+    case getType(deleteUselessOrganizations):
```

Comment:
`entities.organizations.all` redux state is no more used by the application. [This](https://github.com/pagopa/io-app/pull/1540) pr removes it from the store. You could avoid these changes to avoid conflicts when both the pr will be merged